.golangci.yml

```diff
@@ -11,6 +11,7 @@ linters:
   - gosimple
   - govet
   - ineffassign
+  - interfacer
```

Comment:
https://github.com/mvdan/interfacer appears to be archived. should we find a currently maintained version of this? is the archived version still considered the recommended one?

Reply:
yeah... we're limited to what `golangci-lint` provides. in some cases they maintain their own forks:
https://github.com/golangci/golangci-lint/blob/master/go.mod#L13-L29
...and in other cases they depend directly on other linters:
https://github.com/golangci/golangci-lint/blob/master/go.mod#L57

i think it's a worthwhile trade-off to defer to `golangci-lint`  on linter selection, as it's fairly well-supported as a whole, and also quite fast compared to running each linter separately.